README.md

```diff
@@ -1,45 +1,41 @@
 # Vue A11yDialog [![Build Status](https://travis-ci.org/morkro/vue-a11y-dialog.svg?branch=master)](https://travis-ci.org/morkro/vue-a11y-dialog)
 
-This is a Vue.js wrapper component for [`a11y-dialog@5.4.1`](https://github.com/edenspiekermann/a11y-dialog) ([**demo**](https://codesandbox.io/s/rj20wr1kpp)).
+This is a Vue.js wrapper component for [`a11y-dialog@7.3.0`](https://github.com/KittyGiraudel/a11y-dialog) ([**demo**](https://codesandbox.io/s/rj20wr1kpp)).
 
-**Vue 2 and 3**
-
-The newest major release of this package comes with Vue 3 support and removes the dependency to [`portal-vue@2.1.7`](https://github.com/LinusBorg/portal-vue). If you still need to support Vue 2, you can stay on version [`0.5.2`](https://github.com/morkro/vue-a11y-dialog/tree/0.5.2).
-
-- [Install](#install)
+- [Installation](#installation)
 - [Usage](#usage)
   - [Multiple dialogs](#multiple-dialogs)
 - [API](#api)
 - [Events](#events)
 - [Slots](#slots)
 - [Server-side Rendering](#server-side-rendering)
 
-## Install
+## Installation
+
+This library supports both Vue 3 and Vue 2. However, active maintenance is focused on Vue 3. If you still need to support Vue 2, you can stay on version [`0.5.2`](https://github.com/morkro/vue-a11y-dialog/tree/0.5.2).
+
+**Vue 3**
 
 ```bash
 npm install vue-a11y-dialog
 ```
 
-## Usage
+**Vue 2**
 
-In your `main.js` application file, install the component:
-
-### Vue 2
-
-```js
-import VueA11yDialog from 'vue-a11y-dialog'
-
-Vue.use(VueA11yDialog)
+```bash
+npm install vue-a11y-dialog@0.5.2
 ```
 
-### Vue 3
+## Usage
+
+In your `main.js` application file, install the component:
 
 ```js
 import { createApp } from 'vue'
-import VueA11yDialog from 'vue-a11y-dialog'
+import A11yDialog from 'vue-a11y-dialog'
```

Comment:
From our discussion figuring out how to indeed use the default export in esm, I believe what's here would in fact work for the UMD build but might be nice to add a note for ESM you need to do `A11yDialog.install(app)` pattern

Reply:
More completely for esm this worked for me:

```js
import { createApp } from 'vue'
import Demo from './Demo.vue'
import A11yDialog from '../dist/vue-a11y-dialog.esm.js'
import './demo.css'
const app = createApp(Demo)
A11yDialog.install(app)
app.mount('#app')
```